doc/swagger.json

```diff
@@ -353,8 +353,8 @@
                     {
                         "name": "overwriteTest",
                         "in": "formData",
-                        "description": "Boolean flag that indicates if already existed tests and all related items are removed in a case they have the same title and class as uploaded tests. Can be used to replace existing test and avoid creating a new one with the same title and class.",
-                        "type": "boolean",
+                        "description": "Flag that indicates if already existed tests and all related items are removed in a case they have the same title and class as uploaded tests. Can be used to replace existing test and avoid creating a new one with the same title and class. 'true' value counts as true, everything else as false",
+                        "type": "string",
```

Comment:
There are multiple boolean API parameters, if you change how it is processed, I would say lets stay consistent and update the documentation/processing for all of them.
There is even another api endpoint with same parameter `overwriteTest` that would be treated differently

Reply:
Updated another `overwriteTest` parameter.
Processing of all other boolean parameters were not been changed. Actually there is an issue in documenting such parameters because all of them process like `true` `false` string, not boolean. I didn't change the doc because they are not connected to my ticket.

Reply:
Hmm, if you use `filter_var(..., FILTER_VALIDATE_BOOLEAN)`, then not only `"true"` is counted as true, but also `"On"`, `"1"`, `"Yes"`, if this [documentation comment](https://www.php.net/manual/en/function.filter-var.php#121263) is still valid. If all other `boolean` API parameters are treated the same, I'd say lets keep the documentation the same 🤔